device-types/HPE/ProLiant_ML110_Gen9.yaml

```diff
@@ -0,0 +1,41 @@
+---
+manufacturer: HPE
+model: ProLiant ML110 Gen9
+slug: hpe_ml110_gen9
+u_height: 3
+is_full_depth: true
+console-ports:
+  - name: VGA
+    type: other
+  - name: Front USB1
+    type: usb-a
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```

Comment:
This is not the intendde use case for the console port module.  Please remove

Reply:
changed all USB ports to front / rear ports